playbooks/rest_objects/nfsServer/delete.yml

```diff
@@ -1,18 +1,19 @@
 ---
 - hosts: localhost
   vars_files:
-    - ../../group_vars/all
+    - ../../../group_vars/all
   tasks:
     - name: Get nfsServer
-      nfsServer:
+      commonGetPost:
```

Comment:
I think, that someteimes it's better to use get requests from client  (REST object) modules, like:

    nfsServer:
      ...
      get:
         ...

Reply:
What kind of sometimes are you talking about?

Reply:
Name a reason for it. The sdk allows you to do that, and it should be mentioned in the docs, but both methods are equally good.